src/ai_client.c

```diff
@@ -3,6 +3,8 @@
  *
  * Copyright (C) 2009-2011 Keldon Jones
  *
+ * Source file modified by J.-R. Reinhard, October 2016.
```

Comment:
A better solution would be to have a separate communication/protocol version. We can start with #define COMM_VERSION "0.9.4", send that during login. The server should then only accept logins matching its COMM_VERSION.